src/test/java/org/jabref/logic/importer/fetcher/DoiResolutionTest.java

```diff
@@ -38,13 +38,13 @@ void linkWithPdfInTitleTag() throws IOException {
     @Test
     void linkWithPdfStringLeadsToFulltext() throws IOException {
         entry.setField(StandardField.DOI, "10.1002/acr2.11101");
-        assertEquals(Optional.of(new URL("https://onlinelibrary.wiley.com/doi/epdf/10.1002/acr2.11101")), finder.findFullText(entry));
+        assertEquals(Optional.of(new URL("https://onlinelibrary.wiley.com/doi/pdf/10.1002/acr2.11101")), finder.findFullText(entry));
     }
 
     @Test
-    void multipleLinksWithSmallEditDistanceLeadToFulltext() throws IOException {
```

Comment:
Why was this test removed?

Reply:
Edit distance functionality was removed. Actually it was the reason for the bug.

Reply:
The paper used at that test case is used at another test case.

Reply:
Ah ok, now I see. Thanks!